tests/gem5/configs/arm_boot_exit_run.py

```diff
@@ -199,15 +199,15 @@
 
 # Set the Full System workload.
 board.set_kernel_disk_workload(
```

Comment:
Does it make sense to pack this up into a workload?

Reply:
Maybe, but I also like to have some "raw" set_workload examples around as well. Both are helpful.

Either way, I think I can do this in another PR if required?